build-tools/xaprepare/xaprepare/Scenarios/Scenario_PrepareExternal.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Xamarin.Android.Prepare
+{
+	[Scenario (isDefault: false)]
+	partial class Scenario_PrepareExternal : ScenarioNoStandardEndSteps
+	{
+		public Scenario_PrepareExternal ()
+			: base ("PrepareExternal", "Prepare external submodules", Context.Instance)
+		{}
+
+		protected override void AddSteps (Context context)
+		{
+			Steps.Add (new Step_PrepareExternal ());
+			Steps.Add (new Step_PrepareExternalJavaInterop ());
+
+			// disable installation of missing programs...
+			context.SetCondition (KnownConditions.AllowProgramInstallation, false);
+
+			// ...but do not signal an error when any are missing
+			context.SetCondition (KnownConditions.IgnoreMissingPrograms, true);
+		}
```

Comment:
what is the reason for disabling the installation and error reporting here?

Reply:
Good question.  Likely a bit of cargo-cult programming & copying existing code...

Reply:
Removed the disabling, and it still works!